db/c_test.c

```diff
@@ -1825,6 +1828,7 @@ int main(int argc, char** argv) {
     CheckCondition(1 == rocksdb_options_get_is_fd_close_on_exec(copy));
     CheckCondition(1 == rocksdb_options_get_skip_log_error_on_recovery(copy));
     CheckCondition(18 == rocksdb_options_get_stats_dump_period_sec(copy));
+    CheckCondition(600 == rocksdb_options_get_stats_persist_period_sec(copy));
```

Comment:
I suppose there should be 5 here, not 600.

Reply:
done